cmd/flow-aggregator/config.go

```diff
@@ -51,4 +51,31 @@ type FlowAggregatorConfig struct {
 	// Provide format for records sent to the configured flow collector. Supported formats are IPFIX and JSON.
 	// Defaults to "IPFIX"
 	RecordFormat string `yaml:"recordFormat,omitempty"`
+	// KafkaParams contains all configuration parameters required to export flow
+	// records to a Kafka broker.
+	KafkaParams KafkaConfig `yaml:"kafkaParams,omitempty"`
+}
+
+type KafkaConfig struct {
+	// Provide the Kafka broker address to send the aggregated flow records in the
+	// given protobuf format. Kafka flow export is only enabled
+	// if the broker address is provided.
+	// Defaults to ""
+	BrokerAddress string `yaml:"kafkaBrokerAddress,omitempty"`
```

Comment:
Field names in this file are not consistent with those defined in conf file. I think that's why the e2e tests do not pass.